docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,34 +29,164 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
 
 ## [Output](#output)
 
- The OncoTree Mapper Tool will add a new column called `ONCOTREE_SUGGESTIONS` containing suggestions for OncoTree codes if a direct mapping was not available.
+ The OncoTree Mapper Tool will automatically replace the value in the `ONCOTREE_CODE` column with the mapped code if available. The tool will also add a new column called `ONCOTREE_SUGGESTIONS` containing suggestions for OncoTree codes if one or more nodes could not be direclty mapped. The `ONCOTREE_SUGGESTIONS` column formats its suggestions differently depending on the mapping results. Possible suggestion formats and corresponding examples are shown below.
+ 
+ ### 1. Unambiguous Direct Mappings
+ Unambiguous direct mappings occur when an oncotree code maps directly to a single code in the target version. In this case, the `ONCOTREE_SUGGESTIONS` column will be left blank, and the mapped code will be automatically placed in the `ONCOTREE_CODE` column. Unambiguous direct mappings are checked for addition of more granular nodes; to see how this may affect the `ONCOTREE_SUGGESTIONS` column formatting, please refer to the section on [More Granular Nodes Introduced](#4-more-granular-nodes-introduced).
```

Comment:
the oncotree_code column will not be filled in if there are more granular nodes .... but we can correct this in revision